### DISCUSSION TITLE
Allow separate issues tab on forks of a repo?

### DISCUSSION THREAD

**Discussion by ffd8** [community/community#11776] [Projects and Issues]
Dear Github Team

I just ran into an interesting situation... there's a [Processing library for using openCV](https://github.com/atduskgreg/opencv-processing) which has run into issues with Processing's 4.0 beta... a [Stackoverflow thread](https://stackoverflow.com/questions/66285416/processing-4-error-warning-illegal-reflective-access-by-gab-opencv-opencv/68493248) points people to a [fork of that library](https://github.com/jaegonlee/opencv-processing). So at the moment, development is continuing and the community are looking at a fork, which may eventually find it's way to the main project, but in the meantime, there's no way to discuss additional issues/ideas for that person's fix, since the 'issues' tab is missing on a fork. Sorry if this is just unique to this repo/fork and normally it's there.. or have you considered allowing totally separate issues to be written on forks?

Cheers!

**Comment by derMart** [comment]
Yes, please allow issues on forks. This is one of the main drawbacks of Github at the moment in my opinion. Another scenario is when a project is not maintained anymore (even worse if the project gets archived so issues are read only). A fork then has no option to track its issues, very bad situation, seen a few times already.

**Comment by tyler2cr** [comment]
+1 for this feature request. It might be a prerequisite for adding support for forks in projects: https://github.com/github/feedback/discussions/14977

**Comment by syntaxsdev** [comment]
+1 also bump

**Comment by sakul-the-one** [comment]
+1
